stable/rbac-manager/CODEOWNERS

```diff
@@ -1 +1 @@
-* @robscott @davekonopka @sudermanjr
```

Comment:
FYI, I left only the chart maintainers here, I just happen to be both in this case. _Dave will be included by the top-level.

I left my name here in case I ever stop being a CODEOWNER of the entire repo.